ReleaseProcessScript/tools/Core/main_functions.ps1

```diff
@@ -318,25 +316,25 @@ function Release-RC ()
     
   if ([string]::IsNullOrEmpty($Ancestor) )
   {
-    $Ancestor = Get-Ancestor "develop", "support/v"
+    $Ancestor = Get-Ancestor "develop", "hotfix/v"
   }
     
   $CurrentBranchname = Get-Current-Branchname
   $LastVersion = Parse-Version-From-ReleaseBranch $CurrentBranchname
 
   $CurrentVersion = Find-Next-Rc $LastVersion
 
-  if ($Ancestor -eq "develop")
+  if ($Ancestor -eq "develop" -or $Ancestor.StartsWith("release/"))
```

Comment:
Is this part of the current issue?

Reply:
Ah, yes it could be part of the "fix incorrect tests" one. I frankly can't remember